tests/suit/ova_test.go

```diff
@@ -32,6 +32,7 @@ var _ = Describe("[level:component]Migration tests for OVA provider", func() {
 		secret, err := utils.CreateSecretFromDefinition(f.K8sClient, utils.NewSecretDefinition(
 			map[string]string{
 				"createdForProviderType": "ova",
+				"createdForResourceType": "providers",
```

Comment:
@bkhizgiy that took me quite some time yesterday, to realize why that's needed - apparently OVA providers **must** be processed by the secret mutator since we validate their secrets have the `insecureSkipVerify` property - we don't have to force having it since we should support handling existing providers that don't have that setting, could you please change that?

Reply:
@bkhizgiy I posted https://github.com/kubev2v/forklift/pull/703 for this, please take a look

Reply:
sorry I've missed the previous comment from yesterday, I just comment on the PR, so yes we can entirely drop it, I will also add a fix for the mutator so this property wont be set on the secret (but this part is not a blocker).